doc-content/enterprise-only/admin-and-config/business-central-settings-security-provider-con.adoc

```diff
@@ -0,0 +1,13 @@
+[id='business-central-settings-security-provider-con']
+= Security provider
+
```

Comment:
This chapter needs some restructuring for clarity and some language modifications. Some of the paragraphs need to be moved around in order for the whole section to make sense to the user. For example: since this is a chapter about Security Providers, move the definition and examples of security providers to the beginning. Move the rest of the paragraphs to further describe what and how they are used.